server.c

```diff
@@ -3580,6 +3583,7 @@ handle_tcp_reading(int fd, short event, void* arg)
 	ssize_t received;
 	struct event_base* ev_base;
 	struct timeval timeout;
+	uint32_t now;
```

Comment:
```suggestion
	uint32_t now = 0;
```
Perhaps this is useful